frontend/src/pages/Chapters.jsx

```
@@ -75,8 +75,15 @@ const Chapters = () => {
 
       {chapters.map((chapter) => (
         <div
-          onClick={(e) => navigate(`/chapters/${chapter._id}`)}
+
           key={chapter._id}
+          tabIndex={0}
```

Biome (1.9.4)

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)